libdevcore/JSON.h

```diff
@@ -23,22 +23,31 @@
 #pragma once
 
 #include <json/json.h>
+#include <sstream>
 
 namespace dev
 {
 
 /// Serialise the JSON object (@a _input) with indentation
 inline std::string jsonPrettyPrint(Json::Value const& _input)
 {
-	return Json::StyledWriter().write(_input);
+	std::stringstream stream;
+	Json::StreamWriterBuilder builder;
+	builder.settings_["indentation"] = "  ";
+	std::unique_ptr<Json::StreamWriter> writer(builder.newStreamWriter());
+	writer->write(_input, &stream);
+	return stream.str();
 }
 
 /// Serialise the JSON object (@a _input) without indentation
 inline std::string jsonCompactPrint(Json::Value const& _input)
 {
-	Json::FastWriter writer;
-	writer.omitEndingLineFeed();
-	return writer.write(_input);
+	std::stringstream stream;
```

Comment:
Please combine this code with the one above.

Reply:
Sure!